janitor/functions/concatenate_columns.py

```diff
@@ -52,7 +52,10 @@ def concatenate_columns(
         raise JanitorError("At least two columns must be specified")
 
     df[new_column_name] = (
-        df[column_names].astype(str).fillna("").agg(sep.join, axis=1)
+        df[column_names]
+        .astype(str)
+        .replace(["NaT", "nan", "<NA>"], "")
+        .agg(sep.join, axis=1)
```

Comment:
We might want an argument here to toggle between old and new behaviours. Would you be open to doing so?

Reply:
I'm sorry, the translation software I used may not have described it clearly。
For ignoring null values, my idea comes from the implementation of Excel, feeling that the implementation of Excel is more in line with the actual use
https://support.microsoft.com/en-us/office/textjoin-function-357b449a-ec91-49d0-80c3-0e8fc845691c

Reply:
It is my understanding that if null values are not ignored, then this is more reasonable.